### PR TITLE
Simple test for TR.

### PR DESCRIPTION
**Pull request by ByzanTine**

### CODE REVIEW

**Comment by ByzanTine** [comment]
3 of #23

File: cifar10-pytorch/tr_test.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        # At x = 1 for x^2
+        hess_fn = lambda x: 2 * x
+        grad = torch.Tensor([2.0])
+        delta = 2
+        error_tol = 1e-9
+        max_iters = 250
+        x0 = torch.Tensor([1])
+        x, tr_min, iters = solve_tr_subprob(hess_fn, grad, delta, error_tol,
+                                            max_iters, x0)
+        self.assertTrue(x <= delta, msg='{0}'.format(x))
```
Comment (LinjianMa) [inline cifar10-pytorch/tr_test.py]: What is this line testing? Why is it comparing to 2?

Comment (ByzanTine) [reply cifar10-pytorch/tr_test.py]: Just the resulted x falls in the region defined by the radius (Ideally this should be |x| \le 2)

**Comment by LinjianMa** [comment]
Looks good to me

**Pull request merged**